index.js

```diff
@@ -0,0 +1,119 @@
+var days = [
+    'SUN',
+    'MON',
+    'TUE',
+    'WED',
+    'THU',
+    'FRI',
+    'SAT'
+];
+
+var months = [
+    'JAN',
+    'FEB',
+    'MAR',
+    'APR',
+    'MAY',
+    'JUN',
+    'JUL',
+    'AUG',
+    'SEP',
+    'OCT',
+    'NOV',
+    'DEC'
+];
+
+var monthdays = [31,28,31,30,31,30,31,31,30,31,30,31];
+
+var t = new Date();
+var tdate = t.getDate();
+var tday = t.getDay();
+var tmonth = t.getMonth();
+var tyear = t.getFullYear();  // 앞에 t 붙어있는거 -> 현재 날짜 가져옴.
+
+var d = tdate;
+var m = tmonth;
+var y = tyear;
+var nday = new Date(y,m,d).getDay();
+var arr = [];  // 날짜 저장
+
+function showDays(){
+
+    document.querySelector('.year').innerHTML = y;
+    document.querySelector('.dal').innerHTML = months[m];
+    document.querySelector('.date').innerHTML = d;
+    document.querySelector('.days').innerHTML = days[nday];
+
+    var lastday = monthdays[m];  // 이 달의 마지막 날
+    var firstday = new Date(y,m).getDay(); // 이 달의 첫번째 요일, 요일수만큼 0이 앞에 생긴다.
+    var count = "";
+
+    for (j=0; j<firstday; j++){
```

Comment:
띄어쓰기에 유의해 주세요.
일반적으로 아래와 같이 작성합니다.
```suggestion
    for (j = 0; j < firstday; j++){
```